src/state/index.ts

```diff
@@ -39,7 +39,16 @@ const store = configureStore({
     [routingApi.reducerPath]: routingApi.reducer,
   },
   middleware: (getDefaultMiddleware) =>
-    getDefaultMiddleware({ thunk: true })
+    getDefaultMiddleware({
+      thunk: true,
+      serializableCheck: {
+        isSerializable: (value: any) => {
```

Comment:
it feels odd to have a routing api specific detail at the top level state middleware? are there alternatives to this